play26-core/src/main/scala/play/api/test/ops/AsyncResultExtractors.scala

```diff
@@ -0,0 +1,168 @@
+package play.api.test.ops
+
+import akka.actor.Cancellable
+import akka.stream.{Attributes, ClosedShape, Graph, Materializer}
+import akka.util.ByteString
+import play.api.http.HeaderNames._
+import play.api.http.Status._
+import play.api.libs.json.{JsValue, Json}
+import play.api.libs.streams.Accumulator
+import play.api.mvc._
+import play.twirl.api.Content
+
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+
+/**
+  * Similar to play.api.test.ResultExtractors except that it is designed to work better
```

Comment:
HINT: Use `[[play.api.test.ResultExtractors]]` to create a link to the class.

This can be applied here and several other places in the code.